MediaBrowser.Providers/Subtitles/SubtitleManager.cs

```diff
@@ -349,10 +349,10 @@ public Task DeleteSubtitles(BaseItem item, int index)
         /// <inheritdoc />
         public Task<SubtitleResponse> GetRemoteSubtitles(string id, CancellationToken cancellationToken)
         {
-            var parts = id.Split(new[] { '_' }, 2);
+            var parts = id.Split('_', 2);
 
             var provider = GetProvider(parts[0]);
-            id = parts.Last();
+            id = parts[1];
```

Comment:
Shouldn't this be ^1? The second parameter for split is just the max number of strings. If there's no match it will just be one string, right?